tools/hangar/src/__snapshots__/invalid.test.ts.snap

```diff
@@ -2,126 +2,131 @@
 
 exports[`access_hidden_namespace.w > wing test (--target sim) - invalid > stderr 1`] = `
 "Compilation failed with 1 error(s)
-Error at ../../../../examples/tests/invalid/access_hidden_namespace.w:8:5 | \\"core\\" was not brought"
+Error at access_hidden_namespace.w:8:5 | Unknown symbol \\"core\\""
 `;
 
 exports[`bring.w > wing test (--target sim) - invalid > stderr 1`] = `
 "Compilation failed with 1 error(s)
-Error at ../../../../examples/tests/invalid/bring.w:1:1 | Redundant import of \\"std\\""
+Error at bring.w:1:1 | Redundant import of \\"std\\""
+`;
+
+exports[`bring_jsii.w > wing test (--target sim) - invalid > stderr 1`] = `
+"Compilation failed with 1 error(s)
+Error at bring_jsii.w:1:1 | bring \\"\\"jsii-code-samples\\"\\" must be assigned to an identifier"
 `;
 
 exports[`capture_mutables.w > wing test (--target sim) - invalid > stderr 1`] = `
 "Compilation failed with 5 error(s)
-Error at ../../../../examples/tests/invalid/capture_mutables.w:8:10 | Cannot reference 'a' of type 'MutArray<str>' from an inflight context
-Error at ../../../../examples/tests/invalid/capture_mutables.w:10:10 | Cannot reference 's' of type 'MutSet<num>' from an inflight context
-Error at ../../../../examples/tests/invalid/capture_mutables.w:12:10 | Cannot reference 'm' of type 'MutMap<bool>' from an inflight context
-Error at ../../../../examples/tests/invalid/capture_mutables.w:14:10 | Cannot reference 'a_cloned' of type 'MutArray<str>' from an inflight context
-Error at ../../../../examples/tests/invalid/capture_mutables.w:14:19 | Unknown symbol \\"size\\""
+Error at capture_mutables.w:8:10 | Cannot reference 'a' of type 'MutArray<str>' from an inflight context
```

Comment:
The changes to the path names in this diff is because I changed the Parser so it's initialized in `lib.rs` with the source file's `file_name` and not the entire (relative or absolute) path. Definitely something we can change back if it becomes confusing 🙂